AmplifyTestCommon/Models/NonModel/Todo+Schema.swift

```diff
@@ -33,7 +33,7 @@ extension Todo {
       .field(todo.name, is: .required, ofType: .string),
       .field(todo.description, is: .optional, ofType: .string),
       .field(todo.categories, is: .optional, ofType: .embeddedCollection(of: Category.self)),
-      .field(todo.section, is: .optional, ofType: .embedded(type: Section.self)),
+      .field(todo.section, is: .optional, ofType: .customType(type: Section.self)),
```

Comment:
Is this right? Shouldn't it be embedded?

Reply:
I was using this to test that using `.customType` compiles. actually let me add another class entirely and put a comment on Amplify CLI version, and deprecated note

Reply:
updated